apps/wallet-mobile/src/features/Settings/PrivacyMode/PrivacyMode.tsx

```diff
@@ -42,3 +42,17 @@ const parsePrivacyMode = (data: unknown) => {
 
   return isPrivacyMode(parsed) ? parsed : undefined
 }
+
+export const useTogglePrivacyMode = () => {
+  const privacyMode = useReadPrivacyMode()
+  const writePrivacyMode = useWritePrivacyMode()
+  return () => writePrivacyMode(privacyMode === 'SHOWN' ? 'HIDDEN' : 'SHOWN')
+}
+
+export const usePrivacyMode = () => {
+  return {
+    privacyMode: useReadPrivacyMode(),
```

Comment:
I'd recommend to return `isPrivacyOn: privacyMode === 'HIDDEN'`, `isPrivacyOff: privacyMode === 'SHOW'`, the way we storage doesn't matter, it just a boolean to the client.